authors/apps/articles/urls.py

```diff
@@ -14,5 +15,8 @@
     path('articles/<slug>/like/',
          LikeArticleApiView.as_view(), name='likes'),
     path('articles/<slug>/dislike/',
-         DislikeArticleApiView.as_view(), name='dislikes')
+         DislikeArticleApiView.as_view(), name='dislikes'),
+    path('articles/read/<slug:slug>/',
```

Comment:
What's the difference between `<slug:slug>` and `<slug>`

Reply:
I had used <slug: slug> because the detail view was using <slug:  slug> but since the output is still the same I switched to <slug>